source/usage-examples/deleteMany.txt

```diff
@@ -7,13 +7,15 @@ Delete Multiple Documents
 You can delete multiple documents in a collection by using the
 ``DeleteMany()`` method. 
 
-The following example specifies a query filter to the ``DeleteMany()``
-method, which matches documents in the ``movies`` collection where the
-``runtime`` field is greater than *800* and deletes all the matching
-documents. 
+Example
+-------
 
 .. include:: /includes/usage-examples/run-example-tip.rst
 
+The following example matches documents in the ``movies`` collection
+in which the ``runtime`` field is greater than 800 min, deleting all 
```

Comment:
Either use "minutes" or omit the unit of measurement. I think "min" is more confusing that either alternative since it could stand for "minimum".